Plugin/Checkout/Model/MethodMetaData/IconRendererPlugin.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I just noticed that we are not using strict_types anywhere in this module. For consistency, this is being removed.

Reply:
Fixed by my own.